src/main/java/backend/LoginController.java

```diff
@@ -61,7 +61,8 @@ public void getPreviousLoginDetails() {
 
     public boolean attemptLogin() {
 
-        boolean validRepoId = Utility.isWellFormedRepoId(owner, repo) && !username.isEmpty() && !password.isEmpty();
+        boolean validRepoId = RepoId.isWellFormedRepoIdString(owner + "/" + repo)
+                && !username.isEmpty() && !password.isEmpty();
         boolean loginSuccessful = login(username, password);
 
         if (validRepoId && loginSuccessful) {
```

Comment:
Something is not quite right here. Does the validity of the repo ID matters to the login at all?